<?xml version="1.0"?>

<launch>

    <include file="$(find dynamic_obstacle_avoidance_planner)/launch/robot_predictor.launch"/>
    <include file="$(find dynamic_obstacle_avoidance_planner)/launch/dynamic_local_costmap.launch"/>
    <include file="$(find dynamic_obstacle_avoidance_planner)/launch/avoidance_path_planner.launch"/>
    <node pkg="dynamic_obstacle_avoidance_planner" type="base_costmap_tf_publisher" name="base_costmap_tf_publisher"/>

</launch>
